nirc_ehr/resources/queries/dbo/q_drug.sql

```diff
@@ -27,16 +27,4 @@ WHERE (evtEvtGrp.EVENT_GROUP_ID = 31 AND evtEvtGrp.EVENT_ID = 2261) -- Sedation
 --     39	Behavioral Treatment
 --     51	Oral Dose Administration - Alert
 --     54	Fluid Administration
---     65	Sedation
-
-GROUP BY anmEvt.ANIMAL_EVENT_ID,
```

Comment:
I believe this was done to ensure there are no duplicate records, did you verify there are no duplicates?